aioresponses/compat.py

```diff
@@ -5,6 +5,7 @@
 from urllib.parse import parse_qsl, urlencode
 
 from aiohttp import __version__ as aiohttp_version, StreamReader
+from aiohttp.client_proto import ResponseHandler
 from multidict import MultiDict
 from pkg_resources import parse_version
```

Comment:
The package throws the following warning when running locally
```
DeprecationWarning: pkg_resources is deprecated as an API
```
The official documentation suggests the use of [importlib.resources](https://docs.python.org/3/library/importlib.resources.html#module-importlib.resources), [importlib.metadata](https://docs.python.org/3/library/importlib.metadata.html#module-importlib.metadata) as a replacement. More info here: https://setuptools.pypa.io/en/stable/pkg_resources.html

Reply:
@brettdh I'm wondering the same thing :-S

Reply:
I can confirm that all unit tests pass in Python 3.11.4 after changing this line (and one other like it, in `tests/test_aioresponses.py`) to this:

```python
from packaging.version import parse as parse_version
```

Using a private fork for now, but hoping that can get fixed, merged, and released soon.